src/main/java/mekanism/common/tile/component/SideConfig.java

```diff
@@ -0,0 +1,56 @@
+package mekanism.common.tile.component;
+
+import net.minecraft.util.EnumFacing;
+
+import java.util.Observable;
+import java.util.Observer;
+
+public class SideConfig {
+    private byte[] directions = new byte[EnumFacing.VALUES.length];
+    private Observable observable = new Observable();
```

Comment:
setChanged is protected, hence why I ask if it should extend (and that's what Google seemed to indicate also) 

It's still kinda a specific type of observable tho, no? I imagine the reason it's not an interface though is that it existed pre Java 8 and default methods?